lottie/src/main/java/com/airbnb/lottie/model/Font.java

```diff
@@ -9,7 +9,7 @@
 
   private final String family;
   private final String name;
-  private final String style;
+  private String style;
```

Comment:
This mutates the Font file on the composition which could be used across many layers or even reused across multiple animation views.
The specific layer with the callback will need to create its own immutable font to use if it differs from the one here.

Reply:
@gpeal Does it really need to be immutable? Cuz I want to change it dynamically based on the user interests, and as they might change the style multiple times I need to change the font style variable as well.

Reply:
@Luosoha the problem is that this font object will be reused by any other layers that use the same font so even if you set the KeyPath on a specific text layer, it will affect other layers with the same font.

Reply:
@gpeal  Aight I understand it now, but can you finish this PR for me? Sadly I'm not capable of doing the works you are asking me for.